flink-connectors/flink-connector-opensearch/src/main/java/org/apache/flink/streaming/connectors/opensearch/OpenSearchSinkWriter.java

```diff
@@ -121,6 +122,11 @@ protected void submitRequestEntries(
                     "Bulk index of {} documents took {}",
                     requestEntries.size(),
                     bulkResponse.getIngestTookInMillis());
+            List<String> failedEntries = new ArrayList<>();
+            //            for(BulkItemResponse bir: bulkResponse.getItems()){
+            //               if (needReprocess(bir)) {
+            //                   failedEntries.add(bir.)
+            //            }
```

Comment:
not implemented yet?

Reply:
I removed it. New SinkWriter is in sink package!